src/common/markdown.ts

```diff
@@ -15,6 +16,9 @@ export const markdown: MarkdownIt = MarkdownIt({
   },
 });
 
+export const getMentionLink = (username: string): string =>
+  `<a href="${WEBAPP_URL}/${username}" target="_blank" data-mention-username="${username}">${username}</a>`;
```

Comment:
I think we have a function that returns the user profile link. If not please create one

Reply:
Also, I'm not sure about the target blank. Please advise with Nimrod.

Reply:
Let me create one as I searched and none came out.

With the target, in the spec, it was mentioned to target blank but after checking with Nimrod a few minutes ago, he wanted it to be on the same tab. Let me apply the changes.